src/DB/Table.php

```diff
@@ -127,6 +127,28 @@ protected function get_collation(): string {
 		return $this->wpdb->has_cap( 'collation' ) ? $this->wpdb->get_charset_collate() : '';
 	}
 
+	/**
+	 * Checks whether a column exists for the table.
+	 *
+	 * @param string $column_name The column name.
+	 *
+	 * @return bool True if the column exists on the table or False if not.
+	 *
+	 * @since x.x.x
+	 */
+	public function has_column( string $column_name ): bool {
+		// phpcs:disable WordPress.DB.PreparedSQL.NotPrepared
+		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+		$this->wpdb->get_results(
+			$this->wpdb->prepare( "SHOW COLUMNS FROM `{$this->get_sql_safe_name()}` LIKE %s", [ '%' . $this->wpdb->esc_like( $column_name ) . '%' ] )
```

Comment:
Why do we check if the column name contains $column_name rather than checking if the table has exactly that column name? For instance, if we have a table with columns named `daily_budget_low` and `daily_budget_high`, and we want to confirm the existence of the column `daily_budget`, this function would return true which is incorrect.

Reply:
Thanks for catching this, I updated the method so I use a strict where clause